src/nethsec/mwan/__init__.py

```diff
@@ -123,7 +123,7 @@ def __store_member(e_uci: EUci, interface_name: str, metric: int, weight: int) -
 
 def store_rule(e_uci: EUci, name: str, policy: str, protocol: str = None,
                source_address: str = None, source_port: str = None,
-               destination_address: str = None, destination_port: str = None) -> str:
```

Comment:
I'd go with a `true`-`false` approach, e_uci is smart enough to convert the whole thing to te appropriate field type